contacts-manage-contacts.adoc

```diff
@@ -15,7 +15,7 @@ Once you've added a contact, you can type the first few letters of their name in
 . Add any relevant details you want.
 . Click btn:[Save].
 
-NOTE: All added contacts appear under *Contacts*.
+NOTE: Contacts appear under the selected contact folder.
```

Comment:
If the user follows the steps mentioned, then the contacts appeared when he navigates to "Contacts" tab. For folder, user needs to explicitly select the folder to add the contact under it. So IMO, the earlier sentence looks good.

Reply:
Fixed.